src/js/timingDiagram.js

```diff
@@ -61,7 +61,7 @@ function timingDiagram(element, response, workflowName, height) {
             var firstEventStart = null;
             var finalEventEnd = null;
 
-            if(callStatus == "Done" || callStatus == "Failed" || callStatus == "RetryableFailure") {
+            if(callStatus == "Done" || callStatus == "Failed" || callStatus == "RetryableFailure" || callStatus == "Aborted" || callStatus == "Bypassed") {
```

Comment:
For next time, I'd recommend making the options an array and using `indexOf()` or similar.